sql/nm_spawn_points.sql

```diff
@@ -6976,6 +6976,11 @@ INSERT INTO `nm_spawn_points` VALUES (17203216,21,-156.857,0.458,485.462);
 INSERT INTO `nm_spawn_points` VALUES (17469747,0,169.270,8.028,-110.580);
 INSERT INTO `nm_spawn_points` VALUES (17469747,1,155.792,8.334,-124.494);
 INSERT INTO `nm_spawn_points` VALUES (17469747,2,169.525,11.964,-128.288);
+INSERT INTO `nm_spawn_points` VALUES (17293485,0,-341.691,-21.0,39.305);
```

Comment:
I know most of the rows do not have it, but please add a comment with the mob name just above these so we know which NM it is in the future.

Reply:
Sure, I will also add a comment to label the spawn point that is an oddity that I mentioned in the PR.